tools/downloader/README.md

```diff
@@ -13,6 +13,8 @@ based on configuration files in the models' directories.
 * `quantizer.py` (model quantizer) quantizes full-precision models in the IR
   format into low-precision versions using Post-Training Optimization Toolkit.
 
+* `networks_compiler.py` compile models in the IR format into executable network blobs.
```

Comment:
Why _networks_ compiler? All of the tools work with networks.

Reply:
I thought just ` compiler` would be too general. But ok, I'll rename.